easybuild/easyconfigs/l/LAMMPS/LAMMPS-23Jun2022-foss-2021b-kokkos.eb

```diff
@@ -37,7 +37,11 @@ dependencies = [
     ('gzip', '1.10'),
     ('cURL', '7.78.0'),
     ('HDF5', '1.12.1'),
-    ('tbb', '2020.3'),
+]
+# TBB is an optional dependency when building on Intel arch
+if ARCH == 'x86_64':
+    dependencies += [('tbb', '2020.3'), ]
```

Comment:
can you move the `tbb` dep to the bottom of the list instead of splitting the list in 2?

Reply:
Yes that seems to work. So I changed it.